internal/backup.go

```diff
@@ -108,49 +124,31 @@ func UploadDto(folder storage.Folder, dto interface{}, path string) error {
 	return folder.PutObject(path, r)
 }
 
-func (backup *Backup) CheckExistence() (exists bool, storage string, err error) {
-	exists, storage, err = backup.SentinelExists()
+func (backup *Backup) CheckExistence() (bool, error) {
+	exists, err := backup.SentinelExists()
 	if err != nil {
-		return false, "", errors.Wrap(err, "failed to check if backup sentinel exists")
+		return false, errors.Wrap(err, "failed to check if backup sentinel exists")
 	}
-	return exists, storage, nil
+	return exists, nil
 }
 
 // AssureExists is similar to CheckExistence, but returns
 // an error in two cases:
 // 1. Backup does not exist
 // 2. Failed to check if backup exist
-func (backup *Backup) AssureExists() (storageName string, err error) {
-	exists, storageName, err := backup.CheckExistence()
+func (backup *Backup) AssureExists() error {
+	exists, err := backup.CheckExistence()
 	if err != nil {
-		return "", err
+		return err
 	}
 	if !exists {
-		return "", NewBackupNonExistenceError(backup.Name)
+		return NewBackupNonExistenceError(backup.Name)
 	}
-	return storageName, nil
-}
-
-func GetBackupByName(backupName, subfolder string, folder storage.Folder) (backup Backup, storageName string, err error) {
```

Comment:
I moved this to `backup_util.go` because it's similar to other functions `GetLatestBackup` and `GetSpecificBackup` that are stored there